src/panels/calendar/ha-panel-calendar.js

```diff
@@ -0,0 +1,282 @@
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+class HaPanelCalendar extends LocalizeMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style include="iron-flex ha-style">
+        .content {
+          padding: 16px;
+          @apply --layout-horizontal;
+        }
+
+        ha-big-calendar {
+          min-height: 500px;
+          min-width: 100%;
+        }
+
+        #calendars {
+          padding-right: 16px;
+          width: 15%;
+        }
+
+        div.all_calendars {
+    ￼     height: 20px;
+    ￼     text-align: center;
+        }
+
+        span.calendar_color {
+          width: 15px;
+          height: 5px;
+     ￼    margin: 0px 10px 2px 0px;
+     ￼    display: inline-block;
+          background-color: #3174ad;
+        }
+     
+        span.calendar_color_spacer {
+          width: 15px;
+        }
+
+        .iron-selected {
+          background-color: #e5e5e5;
+          font-weight: normal;
+        }
+
+        paper-dialog > div.eventContent {
+          margin: 0px 5px 15px 5px;
+        }
+
+        paper-icon-item {
+          padding: 0px; 
+        }
+
+        paper-item-body.start {
+            padding: 0px 30px 0px 0px;
+        }
+
+        paper-dialog {
+            padding: 0px;
+        }
+
+        paper-dialog > h2 {
+            margin: 0;
+            padding: 20px;
+            background-color: #03A9F4;
+        }
+      </style>
+
+    <app-header-layout has-scrolling-region>
+      <app-header slot="header" fixed>
+        <app-toolbar>
+          <ha-menu-button narrow='[[narrow]]' show-menu='[[showMenu]]'></ha-menu-button>
+          <div main-title>[[localize('panel.calendar')]]</div>
+        </app-toolbar>
+      </app-header>
+
+      <div class="flex content">
+        <div id="calendars" class="layout vertical wrap">
+          <paper-card heading="Calendars">
+            <div class="all_calendars">
+              <paper-checkbox id="all_calendars" on-change="checkAll" checked>All calendars</paper-checkbox>
+            </div>
+            <paper-listbox id="calendar_list" multi on-selected-items-changed="_fetchData" selected-values="{{selectedCalendars}}" attr-for-selected="item-name">
+              <template is="dom-repeat" items="[[calendars]]">
+                <paper-item item-name="[[item.entity_id]]">
+                  <span class="calendar_color" style="background-color: [[item.color]]"></span>
+                  <span class="calendar_color_spacer"></span>
+                  [[item.name]]
+                </paper-item>
+              </template>
+            </paper-listbox>
+          </paper-card>
+        </div>
+        <div class="flex layout horizontal wrap">
+          <ha-big-calendar hass='[[hass]]' events="[[items]]"></ha-big-calendar>
+        </div>
+      </div>
+    </app-header-layout>
+
+    <paper-dialog id="eventDetails" entry-animation="fade-in-animation" exit-animation="fade-out-animation" >
```

Comment:
Selected event dialog should be it's own component. But please ,let's remove it for the first version. It's already big enough.